package.json

```diff
@@ -147,6 +147,7 @@
     "storybook": "storybook dev -p 6006",
     "start": "storybook dev -p 6006",
     "build-storybook": "storybook build",
+    "postbuild-storybook": "install -d ./storybook-static/.nojekyll",
```

Comment:
We use `install -d` here because there are cases where this target might not exist (like when we publish to Chromatic where the `build-storybook` command is passed a custom output dir). In this case, we don't worry about it, but don't want the command to fail because the dir is missing.